src/lib/util/options.mli

```diff
@@ -55,14 +55,27 @@ type interpretation =
   | ILast        (** Compute only the last interpretation just before
                      returning SAT/Unknown *)
 
+type smtlib2_version =
+  [ `Latest
+  (** Latest version of the SMT-LIB standard. *)
+  | `V2_6
+  (** The SMT-LIB standard: Version 2.6
+      https://smt-lib.org/papers/smt-lib-reference-v2.6-r2021-05-12.pdf *)
```

Comment:
We should use the link syntax of `odoc` here: https://ocaml.github.io/odoc/odoc_for_authors.html#links-and-references

Reply:
This was written as is because I intended it to render as the full URL. I can make it a text link.

Reply:
I thought it wasn't clickable without this syntax. I'm ok with both solution ;)